vagrant/.gitignore

```diff
@@ -1 +1,3 @@
-.vagrant
\ No newline at end of file
+.vagrant
+*.box
+Berksfile.lock
```

Comment:
The Berksfile works similarly to a Gemfile. Using the cookbook’s metadata, plus any additional entries in the Berksfile, Berkshelf will build a dependency tree of cookbooks that you depend upon, and output that to a Berksfile.lock.

Certainly, the Berksfile.lock can be committed and tracked, but it makes updating any dependency cookbooks harder, as you have to run `berks update` and then commit the changes to Berksfile.lock.

@eherot & I tend to prefer to make sensible version constraints of dependency cookbooks in the metadata or Berksfile, and trust that those cookbooks will do the right thing (e.g., obey SemVer and only issue breaking changes with a bump to the major number).

In this way, you can always be working with the most recent bugfix and feature releases of dependency cookbooks.